src/use-css.hook.js

```diff
@@ -17,28 +17,29 @@ export function useCss(css, overrideNamespace) {
 
   function useStyleElement() {
     useLayoutEffect(() => {
-      const newStyleElement = getStyleElement(styleElement, isPostCss, css, namespace)
+      const newStyleElement = getStyleElement(isPostCss, css, namespace, false)
       setStyleElement(newStyleElement)
 
       return () => {
         if (--styleElement.kremlings === 0) {
+          const rawCss = isPostCss ? css.styles : css
           document.head.removeChild(styleElement)
-          delete styleTags[css]
+          delete styleTags[rawCss]
```

Comment:
For some reason using `css` caused it to fail in production react build (but not development), but using `rawCSS` it doesn't